statik/styles/_layout.scss

```diff
@@ -19,6 +19,7 @@ p {
   font-weight: 400;
   margin: 0;
   padding: 0;
+  max-width: 80ch;
```

Comment:
@eubond Please add the CSS properties in alphabetical order.

Thanks for your contribution! 😉